pkgdiff.pl

```diff
@@ -306,48 +310,57 @@ ()
 
   -template
       Create XML-descriptor template ./VERSION.xml
-      
+
   -extra-info DIR
       Dump extra info to DIR.
-      
+
   -tmp-dir DIR
       Use custom temp directory.
-  
+
   -hide-unchanged
       Don't show unchanged files in the report.
 
   -debug
       Show debug info.
-  
+
   -name NAME
       Set name of the package to NAME.
-  
+
   -title TITLE
       Set name of the package in the title of the report to TITLE.
-  
+
   -vnum1 NUM
       Set version number of the old package to NUM.
-  
+
   -vnum2 NUM
       Set version number of the new package to NUM.
-  
+
   -links-target TARGET
       Set target attribute for links in the report:
         _self
         _blank (default)
-  
+
   -list-added-removed
       Show content of added and removed text files.
-  
+
   -skip-subarchives
       Skip checking of archives inside the input packages.
-  
+
   -skip-pattern REGEX
       Don't check files matching REGEX.
-  
+
   -d|-directories
       Compare directories instead of packages.
 
+  -check-byte-code
+      When comparing Java classes, also check for byte code changes.
+
+  -verbose
```

Comment:
Name of this option is too common. Please change it.

Reply:
:+1: